doc/api/process.md

```diff
@@ -958,6 +958,32 @@ $ bash -c 'exec -a customArgv0 ./node'
 'customArgv0'
 ```
 
+## `process.argv1`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* {string | undefined}
+
+The `process.argv1` property stores a read-only copy of the original value of
+`argv[1]` passed when Node.js starts.
```

Comment:
Can we clarify what value it takes within worker?

Reply:
I added a note. It’s undefined because of https://github.com/nodejs/node/blob/3f1936f698574fc9de45ec9c068b2982c317dee8/lib/internal/process/pre_execution.js#L64.